index.md

```diff
@@ -65,9 +65,15 @@ SoCraTes UK is a **true Software Craftsmanship gathering**. Don't miss out.
 
 ### Where is it taking place?
 
-The retreat will be hosted at [The Fish Hotel][Venue] within Farncombe Conference Centre, a 300+ acre conference and hotel estate in the [Cotswolds][], one of the most beautiful places in the English countryside.
+A leading conference and meeting venue in Surrey, [Wotton House][Wotton] is surrounded by 13 stunning acres of beautifully landscaped grounds, delivering the most tranquil setting and backdrop for your conference or event. 
```

Comment:
Please rewrite to fit more with our writing style or male clear this is citing their website